packages/erxes-ui/src/utils/core.tsx

```
@@ -430,74 +548,74 @@
+  return `${value}`.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
 }
 
 export function numberParser(value, fixed) {
-  if (value === '-') {
-    return '-';
+  if (value === "-") {
+    return "-";
   }
-  if (RegExp('-', 'g').test(value)) {
-    value = value.replace(RegExp('-', 'g'), '');
+  if (RegExp("-", "g").test(value)) {
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)